src/main/java/com/mongodb/kafka/connect/source/schema/BsonDocumentToSchema.java

```diff
@@ -141,10 +142,13 @@ private static Schema combinedSchema(final Schema firstSchema, final Schema seco
                     combinedSchema(firstSchema.valueSchema(), secondSchema.valueSchema()))
                 .name(firstSchema.name())
                 .optional();
+        logIncompatibleSchemasIf(
```

Comment:
What is the effect of moving the location of the logging around like this?  I can't visualize it.

Reply:
This outputs the whole array rather than just the individual incompatible values.